test/screenshot/webpack.config.js

```diff
@@ -5,6 +5,7 @@ const floatingLabelBundles = require('./floating-label/webpack.config.js');
 const lineRippleBundles = require('./line-ripple/webpack.config.js');
 const materialIconBundles = require('./material-icon/webpack.config.js');
 const notchedOutlineBundles = require('./notched-outline/webpack.config.js');
+const textFieldHelperTextBundles = require('./text-field/helper-text/webpack.config.js');
```

Comment:
Can we do that clever require thing in this file too?

Reply:
Don't think so because of the spread operator (eg. `...buttonBundles,`).

Reply:
JK...this works
```js
[
  ...require('./button/webpack.config.js'),
]
```